actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadManagementActor.java

```diff
@@ -247,6 +248,8 @@ private void processBulkBatchEnrollment(Map<String, Object> req, String processI
 
   private void processBulkOrgUpload(Map<String, Object> req, String processId) throws IOException {
 
+    ProjectLogger.log(
+        "BulkUploadManagementActor:processBulkOrgUpload method start.", LoggerEnum.INFO);
```

Comment:
From:
"BulkUploadManagementActor:processBulkOrgUpload method start."

To:
"BulkUploadManagementActor: processBulkOrgUpload called."